src/components/ToolbarMenu.js

```diff
@@ -71,11 +79,17 @@ export class ToolbarMenu extends React.Component {
     }
   }
 
+  handleFocus(e) {
+    preventDefault(e);
+    this.setState({ visible: true });
+  }
+
   handleOpen = e => {
     preventDefault(e);
     this.setState({ visible: true });
     this._subscribe();
   };
+
```

Comment:
Yeah for sure, I'll include that in the next commit I make sometime in the next few days when I figure out how to have menus not close when I try to go through their items heh.